doc/release-notes/10943-featured-items.md

```diff
@@ -0,0 +1,10 @@
+CRUD endpoints for Collection Featured Items have been implemented. In particular, the following endpoints have been implemented:
+
+- Create a feature item (POST /api/dataverses/<dataverse_id>/featuredItems)
+- Update a feature item (PUT /api/dataverseFeaturedItems/<item_id>)
+- Delete a feature item (DELETE /api/dataverseFeaturedItems/<item_id>)
+- List all featured items in a collection (GET /api/dataverses/<dataverse_id>/featuredItems)
+- Delete all featured items in a collection (DELETE /api/dataverses/<dataverse_id>/featuredItems)
+- Update all featured items in a collection (PUT /api/dataverses/<dataverse_id>/featuredItems)
```

Comment:
Can we add a list of new JVM settings?

Here's the raw form:

    SCOPE_FEATURED_ITEMS(SCOPE_FILES, "featured-items"),
    FEATURED_ITEMS_IMAGE_MAXSIZE(SCOPE_FEATURED_ITEMS, "image-maxsize"),
    FEATURED_ITEMS_IMAGE_UPLOADS_DIRECTORY(SCOPE_FEATURED_ITEMS, "image-uploads"),

But we should list them as dataverse.whatever.whatever

Update, I think I found them. We can add this:

New settings:

- dataverse.files.featured-items.image-maxsize
- dataverse.files.featured-items.image-uploads

Reply:
Done. I also added a small description for each setting.

Reply:
Awesome, thanks!